tests/benchmarks/micro_benchmarks/test_disk_performance.py

```diff
@@ -504,34 +504,34 @@ def test_disk_performance_result_parsing(self):
   ]
 }
 """
-        jobname_prefix = 'disk_performance:/dev/nvme0n1:rand_read_write'
+        jobname_prefix = '/dev/nvme0n1_rand_read_write'
```

Comment:
Do we have `/dev/` in the metric name? If so, could we remove `/dev/`?

Reply:
Fixed.